src/app/dataset-block/metadata-block/components/event-details/components/set-vocab-event/set-vocab-event.component.html

```diff
@@ -1,5 +1,15 @@
 <div *ngIf="event" class="position-relative border border-1 p-2 rounded-3">
     <span class="position-absolute text-muted group-box-label text-uppercase">Information</span>
+    <div class="mx-4" *ngIf="event.offsetColumn">
+        <app-block-row-data [tooltip]="'Name of the offset column.'" [label]="'Offset column:'">
+            <app-cards-property [data]="[event.offsetColumn]"></app-cards-property>
+        </app-block-row-data>
+    </div>
+    <div class="mx-4" *ngIf="event.operationTypeColumn">
+        <app-block-row-data [tooltip]="'Name of the operation type column.'" [label]="'Operation type column:'">
+            <app-cards-property [data]="[event.operationTypeColumn]"></app-cards-property>
+        </app-block-row-data>
+    </div>
```

Comment:
Could this be implemented in more compact declarative manner? The template for each event has a regular structure, perhaps we can model this as as array of objects and use `ngFor`.

Reply:
Yes, fixed